src/Uno.UI/Controls/BindableSwitchCompat.Android.cs

```diff
@@ -76,13 +76,9 @@ private void OnThumbTintChanged(Brush newValue)
 		{
 			if (newValue is SolidColorBrush asColorBrush)
 			{
-#if __ANDROID_28__
 #pragma warning disable 618 // SetColorFilter is deprecated
 				ThumbDrawable?.SetColorFilter(asColorBrush.ColorWithOpacity, PorterDuff.Mode.SrcIn);
 #pragma warning restore 618 // SetColorFilter is deprecated
-#else
-				ThumbDrawable?.SetColorFilter(new BlendModeColorFilter(asColorBrush.ColorWithOpacity, BlendMode.SrcIn));
-#endif
```

Comment:
Why are we keeping the deprecated overload and removing the non-deprecated one?

Reply:
This particular part was probably incorrect to begin with, as `__ANDROID_28__` is now always defined.